### PR TITLE
0.H changelog

### PR DESCRIPTION
**Pull request by NetSysFire**
#### Summary
None

#### Purpose of change

The release is nigh.

#### Describe the solution

I have been handpicking some changes which I think are significant enough to be highlighted. These are in no way complete and will undergo tweaks as suggestions pour in.

#### Describe alternatives you've considered

Conveniently vanishing and letting someone else do the work of combing through commits.

#### Testing

n/a, this is not even markdown I think.

#### Additional context

The file is buggy in master as experimental and stable changes are completely mixed up, so this will be fun to fix.

### CODE REVIEW

File: data/changelog.txt [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+Furthermore, the map changes as you travel. The farther east you travel, the bigger cities will be. Cities in the west will start to be farther apart. The forest will get more dense the farther north or west you travel.
+Portal storms are no longer engulfing the entire map and will move. A new boss fight related to portal storms, the void spider, can also be found.
+New mod: Mind over Matter - adds detailed psionics to the game, along with a variety of new monsters, scenarios and game mechanics.
+New scenario: Friends to the End. Ever wanted a NPC companion that feels more alive? With Liam you get a friend that cares and helps out during key moments.
+A massive nuclear power plant has been added. Get ready to explore its giant interior with proper gear.
+Accidentally bumping into neutral creatures and angering them is now (mostly) a problem of the past. Just enable safe mode and you will no longer inadvertently slap your clingy pet gracken.
+3D vision has been improved and you can now see the levels below you automatically, even without 3D FOV enabled. Additionally, you can now throw stuff of ledges to damage creatures below.
```
Comment (XygenSS) [inline data/changelog.txt:14]:
```suggestion
3D vision has been improved and you can now see the levels below you automatically, even without 3D FOV enabled. Additionally, you can now throw stuff off ledges to damage creatures below.
```

File: data/changelog.txt [hunk 2]
```diff
@@ -1,5 +1,22 @@
-# 0.H
+# 0.H (Herbert? Huxley?)
 
+After over a year, we can finally release 0.H, boasting more changes than any previous release.
+
+## Highlights
+
+Oceans! Travel far enough east and you'll be greeted by a sandy shore.
+Furthermore, the map changes as you travel. The farther east you travel, the bigger cities will be. Cities in the west will start to be farther apart. The forest will get more dense the farther north or west you travel.
+Portal storms are no longer engulfing the entire map and will move. A new boss fight related to portal storms, the void spider, can also be found.
```
Comment (harakka) [inline data/changelog.txt:9]: Void spider isn't related to portal storms

Comment (NetSysFire) [reply data/changelog.txt:9]: And how would you word it?

Comment (harakka) [reply data/changelog.txt:9]: If I knew I would have suggested the change.

Comment (Standing-Storm) [reply data/changelog.txt:9]: "In the depths of the earth lurks a Nether horror--fight the new void spider boss if you can find its lair" or something similar?

File: data/changelog.txt [hunk 3]
```diff
@@ -1,5 +1,22 @@
-# 0.H
+# 0.H (Herbert? Huxley?)
 
+After over a year, we can finally release 0.H, boasting more changes than any previous release.
+
+## Highlights
+
+Oceans! Travel far enough east and you'll be greeted by a sandy shore.
+Furthermore, the map changes as you travel. The farther east you travel, the bigger cities will be. Cities in the west will start to be farther apart. The forest will get more dense the farther north or west you travel.
```
[the remark this answers is not in the document]
Comment (NetSysFire) [reply data/changelog.txt:8]: I will not in the changelog as we do not do this in general, see the previous changelogs. Look at #70070 and the related PRs by Erk though if you are interested in the changes.

**Comment by martinrhan** [comment]
For anyone interested in helping to make this changelog, you might check this https://github.com/CleverRaven/Cataclysm-DDA/compare/0.G-branch...0.H-branch.

File: data/changelog.txt [hunk 4]
```diff
@@ -1,4 +1,23 @@
...
+
+After over a year, we can finally release 0.H, boasting more changes than any previous release.
+
+## Highlights
+
+Oceans! Travel far enough east and you'll be greeted by a sandy shore.
+Furthermore, the map changes as you travel. The farther east you travel, the bigger cities will be. Cities in the west will start to be farther apart. The forest will get more dense the farther north or west you travel.
+Portal storms are no longer engulfing the entire map and will move around while active. Additionally, in the depths of the earth now lurks an otherworldly horror: fight the new void spider boss if you can find its lair.
+New mod: Mind over Matter - adds detailed psionics to the game, along with a variety of new monsters, scenarios and game mechanics.
+New scenario: Friends to the End. Ever wanted a NPC companion that feels more alive? With Liam you get a friend that cares and helps out during key moments.
```
Comment (harakka) [inline data/changelog.txt:11]:
```suggestion
Major mod content update: Sky Island has been updated to version 0.3.0, with too many changes to list. Island upgrade,s new items, more missions, more everything.
New scenario: Friends to the End. Ever wanted a NPC companion that feels more alive? With Liam you get a friend that cares and helps out during key moments.
```

File: data/changelog.txt [hunk 5]
```diff
@@ -1,4 +1,23 @@
...
+Guns have been nerfed and now need more skill to be used efficiently. Feral humans are also more challenging now. Don't get too cocky when encountering those.
+A massive nuclear power plant has been added. Get ready to explore its giant interior with proper gear.
+Accidentally bumping into neutral creatures and angering them is now (mostly) a problem of the past. Just enable safe mode and you will no longer inadvertently slap your clingy pet gracken.
+3D vision has been improved and you can now see the levels below you automatically, even without 3D FOV enabled. Additionally, you can now throw stuff off ledges to damage creatures below.
+Certain monsters can now feast on corpses, graze and browse or gobble up your food and crops. That also means in order for an animal to e.g continue producing milk, it must be fed on a regular basis.
+Some monsters can also hide under furniture now.
```
Comment (harakka) [inline data/changelog.txt:17]:
```suggestion
Some monsters can also hide under furniture now.
Animals need to be fed to produce milk, eggs and to reproduce. Animals can graze on grass to feed themselves, and grazed grass regrows after season changes.
```

File: data/changelog.txt [hunk 6]
```diff
@@ -1,4 +1,24 @@
...
+Animals need to be fed to produce milk, eggs and to reproduce. Animals can graze on grass to feed themselves, and grazed grass regrows after season changes.
+You can now find crazed civilians in the cities for the first couple of days. While they can not be turned or "cured" into NPCs, their behavior will make being in a city more dynamic.
+If you have mutations granting you an exoskeleton, you will now molt in regular intervals.
+NPCs can now read E-books and zones can span multiple Z-levels.
```
Comment (db48x) [inline data/changelog.txt:21]: These two changes don’t seem related to each other; split them across two lines?

**Comment by ThePirate42** [comment]
Wasn't the nuclear power plant already in 0.G?

**Pull request merged**